gatherers/offline.js

```diff
@@ -86,6 +88,15 @@ class Offline extends Gather {
       expression: requestPage
     }).then(_ => {
       return Offline.pollForOfflineResponseStatus(driver, 0);
+    }, _ => {
+      // Account for execution errors.
+      return {
+        result: {
```

Comment:
The audit checks only for a 200 code. This code in particular should ensure we don't get JS errors, but you're right we could roll with -1, which in other places in the code indicates an error state. I'll do that.